src/main/java/sirius/biz/tenants/BizInterceptor.java

```diff
@@ -46,11 +55,23 @@ public boolean beforePermissionError(String permission,
         } else {
             ctx.respondWith()
                .template("/templates/wondergem/error.html.pasta",
-                         NLS.fmtr("BizInterceptor.missingPermission").set("permission", permission).format());
+                         NLS.fmtr("BizInterceptor.missingPermission")
+                            .set("permission", getTranslatedPermission(permission))
+                            .format());
         }
         return true;
     }
 
+    private String getTranslatedPermission(String permission) {
```

Comment:
shouldn't we reuse this code elsewhere and maybe make available in the Permissions class?